### PR TITLE
if canvg is passed in, use instead of drawImage

### PR DESCRIPTION
**Pull request by Jephuff**
enables support for IE

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ test('Is loadable using requirejs', function (assert) {
 			'svgAsDataUri': 'function',
 			'svgAsPngUri': 'function',
 			'saveSvgAsPng': 'function',
+			'download': 'function',
+			'prepareSvg': 'function',
+			'saveSvg': 'function',
```
Comment (exupero) [inline test.js:14]: Can you add any docs to the readme about running this test? I'd forgotten someone added it and I don't know how to use it.

**Comment by exupero** [comment]
Glancing over it, looks good. Go ahead and bump the version in package.json to 1.1.0.

Thanks for tackling this. Take a look at the open issues and let me know if there's any that can be closed.

Thanks for documenting the option. I recommend re-writing the Support section of the readme to explain how to use this with IE. It doesn't need to be extensive, but I expect that's where others will look first when troubleshooting IE support.

File: test.js (hunk 1 above)
Comment (Jephuff) [reply test.js:14]: Yep, added

File: README.md [hunk 2]
```diff
@@ -56,7 +56,15 @@ If you want to use TypeScript, necessary [type definitions](https://github.com/m
 - `width` - Specify the image's width. Defaults to the viewbox's width if given, or the element's non-percentage width, or the element's bounding box's width, or the element's CSS width, or the computed style's width, or 0.
 - `encoderType` - A DOMString indicating the image format. The default type is image/png.
 - `encoderOptions` - A Number between 0 and 1 indicating image quality. The default is 0.8
+- `canvg` - If canvg is passed in, it will be used to write svg to canvas. This will allow support for Internet Explorer
+
+### Testing
+
+run tests with `tape`
```
Comment (exupero) [inline README.md]: Thanks. Could you link to the right "tape"?

Comment (Jephuff) [reply README.md]: yep :)

**Comment by Jephuff** [comment]
Looks like there aren't any issues around this. 👍

**Pull request merged**